internal/audit/auditlogtextformatter.go

```diff
@@ -76,19 +77,19 @@ func logInvalidSemVerWarning(sb *strings.Builder, noColor bool, quiet bool, inva
 	}
 }
 
-func logVulnerablePackage(sb *strings.Builder, noColor bool, coordinate types.Coordinate) {
+func logVulnerablePackage(sb *strings.Builder, noColor bool, coordinate types.Projects) {
```

Comment:
I was struggling with the change to `types.Projects`. Made more sense once I saw how the term is used in the `dep` code, e.g.: `dep.Project` - oh yeah, a coordinate really refers to a given "project" that we depend on...Oh the joys of Naming.

Reply:
Yeah I'm not attached to this name TBH. The changes in this PR would merit a new major version of Nancy, so if we want to clean up some naming, let's do it!